lib/livebook/teams/deployment_groups.ex

```diff
@@ -0,0 +1,24 @@
+defmodule Livebook.Teams.DeploymentGroups do
```

Comment:
We should either move the functions here to `Livebook.Teams` here or move the deployment group functions from `Livebook.Teams` to here. For now, keeping it in `Livebook.Teams` is fine, but this will most likely grow in complexity, so I am unsure if we want to move it out now or later.